dist/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>TO-DO LIST</title>
+    <script defer="defer" src="/runtime.bundle.js"></script>
+    <script defer="defer" src="/index.bundle.js"></script>
+    <link href="./index.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div class="main-panel">
+      <h1 class="list-title">Today's To Do</h1>
+      <input id="add-input" class="add-input" placeholder="Add to your list" />
```

Comment:
- [ ] To the right side of the title, there is an icon for sync in the minimalist design. As you have to make an almost similar project, I recommend you to add sync icon.
Your design:
![image](https://user-images.githubusercontent.com/24216198/223470170-e086fb90-fbcd-4271-b3b1-71eba4035dec.png)

Minimalist design:
![image](https://user-images.githubusercontent.com/24216198/223470506-a5dea640-b974-41fb-adba-d24e200df492.png)